### ISSUE TITLE
Fuzzy model name checking

### ISSUE THREAD

**Issue by gjwgit** [mlhubber/mlhub#1]
if model name is not found installed then do a fuzzy check to suggest a close matching name in case of mistyped name. Borrow the functionality from Ubuntu.

**Comment by gjwgit** [comment]
Examples

```
$ ml availble
The command 'availble' is not supported. Did you mean 'available'? [Y|n] 
```

```
$ ml install iras
The model 'iras' was not found on  'https://mlhub.ai/'. Did you mean 'iris'? [Y|n]
```

**Comment by simonzhaoms** [comment]
Implemented in commit 80e2997
